ksqldb-engine/src/test/java/io/confluent/ksql/execution/ExpressionEvaluatorParityTest.java

```diff
@@ -126,22 +126,22 @@ public void shouldDoComparisons() throws Exception {
     assertOrders("ORDERTIME <= 100 AND ITEMID + '-blah' = 'item_id_0-blah'", true);
     assertOrders("ORDERID > 9.5 AND ORDERUNITS < 50.75", true);
     assertOrdersError("ORDERID > ARRAY[0]",
-        compileTime("Cannot compare ORDERID (BIGINT) to ARRAY[0]"),
-        compileTime("Cannot compare ORDERID (BIGINT) to ARRAY[0]"));
+        compileTime("Cannot compare BIGINT to ARRAY<INTEGER>"),
+        compileTime("Cannot compare BIGINT to ARRAY<INTEGER>"));
```

Comment:
Note these are validating the exception message itself, as opposed to the "unlogged details", which are still the same (as verified by other tests).